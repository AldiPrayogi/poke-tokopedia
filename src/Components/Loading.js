import React from 'react';
import {Card, WhiteSpace} from 'antd-mobile';
import Spin from 'antd/es/spin';
import 'antd/es/spin/style/css';
import {loading} from "../Styling/ComponentCSS";

export const Loading =  () => {
    return (
        <div css={loading}>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <Spin size='large'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
            <WhiteSpace size='xl'/>
        </div>
    );
}