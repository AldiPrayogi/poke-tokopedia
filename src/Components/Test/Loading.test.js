import {Loading} from "../Loading";
import {shallow} from 'enzyme';

describe('Loading', () => {
    const wrapper = shallow(<Loading/>);

    it('should render Loading', () => {
        expect(wrapper).toMatchSnapshot();
    });

});