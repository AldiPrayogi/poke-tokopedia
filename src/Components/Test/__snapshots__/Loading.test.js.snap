// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render Loading 1`] = `
<EmotionCssPropInternal
  __EMOTION_LABEL_PLEASE_DO_NOT_USE__="Loading"
  __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
  css={
    Object {
      "map": undefined,
      "name": "1e5c0vr",
      "next": undefined,
      "styles": "
   margin-top: 5%;
   margin-bottom: 5%;
   text-align: center;
   border-radius: 4px;
   width: 100%;
",
      "toString": [Function],
    }
  }
>
  <Spin
    size="large"
    spinning={true}
    wrapperClassName=""
  />
</EmotionCssPropInternal>
`;
